assets/app/views/services.html

```diff
@@ -35,6 +26,37 @@ <h3 id="service-{{service.metadata.name}}">{{service.metadata.name}}</h3>
                 <span ng-show="!$last">, </span>
             </span>
         </div>
+        <div class="table-responsive">
+          <table ng-if="service.spec.ports.length" style="max-width: 650px;">
+            <thead>
+              <tr>
+                <th>Node Port</th>
+                <th role="presentation"></th>
+                <th>
+                  Service Port
+                  <!-- Show portal IP in column header instead of table body at small screen
+                  widths to save space. -->
+                  <span class="visible-xs">({{service.spec.portalIP}})</span>
+                </th>
+                <th role="presentation"></th>
+                <th>Target Port</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="portMapping in service.spec.ports | orderBy:'port'">
+                <td>
+                  <span ng-if="portMapping.nodePort">{{portMapping.nodePort}}</span>
+                  <span ng-if="!portMapping.nodePort" class="text-muted">none</span>
+                </td>
+                <td role="presentation" class="text-muted">&#8594;</td>
+                <td><span ng-if="service.spec.portalIP && service.spec.portalIP !== 'None'">
+                    <span class="hidden-xs">{{service.spec.portalIP}}:</span></span>{{portMapping.port}}</td>
```

Comment:
is this going to look redundant having the portalIP on every line with a lot of ports?